opentelemetry-sdk/src/logs/record.rs

```diff
@@ -198,4 +199,57 @@ mod tests {
             Some(vec![(Key::new("key"), AnyValue::String("value".into()))])
         );
     }
+
+    #[test]
+    fn compare_trace_context() {
+        let span_context = SpanContext::new(
+            TraceId::from_u128(1),
+            SpanId::from_u64(1),
+            TraceFlags::default(),
+            false,
+            TraceState::default(),
+        );
+
+        let span_context_cloned = span_context.clone();
```

Comment:
this test seems to be testing for `SpanContext`, and not `TraceContext`. Or I am missing something?

Reply:
You are right. Just noticed copilot gave me the wrong type when creating the test skelleton and I missed to realize it.
Now should be fixed.